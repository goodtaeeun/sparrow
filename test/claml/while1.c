void f(int x) {
  while (x) {
    x++;
    break;
  }
}
