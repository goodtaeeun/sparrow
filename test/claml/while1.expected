/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "while1.c"
void f(int x) {
  int __cil_tmp2;

  {
    {
#line 2
      while (1) {
      while_continue: /* CIL Label */;
#line 2
        if (!x) {
#line 2
          goto while_break;
        }
#line 3
        x++;
#line 4
        goto while_break;
      }
    while_break: /* CIL Label */;
    }
    return;
  }
}
